### PR TITLE
fix: youtube video thumbnail 0.jpg replaced with default.jpg (120x90)

### PR DESCRIPTION
**Pull request by MAbdurrehman12**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
- [ ] Migrations
  - [ ] Migration is backwards-compatible with current production code
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested
- [ ] Settings
  - [ ] New settings are documented and present in `app.json`
  - [ ] New settings have reasonable development defaults, if applicable
  - [ ] Opened issue for DevOps regarding necessary configuration changes to deployed environments

#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/419

#### What's this PR do?
- Youtube generates multiple thumbnail images, so the image URL can be either:
    - `https://img.youtube.com/vi/<youtube-video-id>/0.jpg ` or  `.../1.jpg`, `.../2.jpg`, `.../3.jpg`, ...
    - The default thumbnail image of size 120x90 is `.../default.jpg`.  
- This PR replaces `.../0.jpg` with `.../default.jpg` so every time, default thumbnail (120x90 size) is pulled.   

#### How should this be manually tested?
- Go to studio
- Add new content/resource with youtube video(s) or update an existing one.
- Publish the course and ensure the thumbnail URL is with  `.../default.jpg not `.../0.jpg`
- Go to the course page -> videos/resources and ensure that the thumbnail images size is uniform and appropriate. ([For example](https://ocwnext-rc.odl.mit.edu/courses/18-06-linear-algebra-spring-2010/video_galleries/video-lectures/))

### CODE REVIEW

File: videos/tasks.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def update_youtube_statuses():
                     set_dict_field(
                         resource.metadata,
                         settings.YT_FIELD_THUMBNAIL,
-                        f"https://img.youtube.com/vi/{video_file.destination_id}/0.jpg",
+                        f"https://img.youtube.com/vi/{video_file.destination_id}/default.jpg",
```
Comment (MAbdurrehman12) [inline videos/tasks.py:154]: `https://img.youtube.com/vi/` and `default.jpg` is being repeated in different places, I think we should move it to constants.
Reviewer can share his/her thoughts.

Comment (mbertrand) [reply videos/tasks.py:154]: Agreed, maybe something like `YT_IMG_PREFIX` and `YT_IMG_SUFFIX`

Comment (MAbdurrehman12) [reply videos/tasks.py:154]: Sure
How about if we keep one constant and use string formatter, something like this:
```
YT_THUMBNAIL_IMG = "https://img.youtube.com/vi/{video_id}/default.jpg"
YT_THUMBNAIL_IMG.format(video_id = video_file.destination_id)
```

What do you suggest?

**Review by mbertrand — APPROVED** [review]
👍

**Comment by pdpinch** [comment]
Will we need to do anything to updated existing data in ocw-studio?

**Pull request merged**